feat: Adding screenshot

CBID2 opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:
This PR adds a screenshot of checkbox in the ["Allowing changes to a pull request branch created from a fork"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) tutorial. This would help readers know where to grant maintainers edit access when making pull requests for open source projects. 
 
### Closes: 

Closes #30407

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):
A screenshot is being added. 
<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

CBID2 committed:
feat: add screenshot

CBID2 committed:
feat: hyperlink screenshot

CBID2 committed:
Merge branch 'main' into adding-screenshot

CBID2 committed:
Merge branch 'main' into adding-screenshot

CBID2 committed:
Merge branch 'main' into adding-screenshot

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md:
```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+![Screenshot of a Pull request. On the right bottom side, a blue, clicked checkbox, labelled "Allow edits and access to secrets by maintainers".](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```
The ! sign must be aligned with the first word in the bullet point (which is "In...the list", so that the ALT text and the screenshot are at the same level as the step or bullet point they relate to.
Also, removing the capitalization of "Pull request" and using US spelling.
Finally, added `is highlighted with an orange outline.` to be consistent with the content model we use.

```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, clicked checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

mchammer01 replied:
Our images or screenshots must be taken with the GitHub default light theme on (for more details, see https://github.com/settings/appearance. You'll need to select "Light default" before taking the screenshot), and also the orange outline has a specific line thickness and a specific round corner measurement.
We may need to provide you with the screenshot ourselves. Let me investigate.

mchammer01 reviewed (commented):
@CBID2 👋🏻 - thanks for improving our docs ✨ 
Made a few comments and will see if I can provide you with the screenshot.

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

mchammer01 commented:
@CBID2 👋🏻 - I'd like to check out your branch to remove your screenshot and add mine, and I don't appear to be able to do it. Can you allow me to check out your branch?

mchammer01 committed:
Merge branch 'main' into adding-screenshot

CBID2 commented:
> @CBID2 👋🏻 - I'd like to check out your branch to remove your screenshot and add mine, and I don't appear to be able to do it. Can you allow me to check out your branch?

Hi @mchammer01! :) Sure you can checkout my branch.

mchammer01 commented:
@CBID2 - do you work locally? Have you pushed your branch to the remote?

CBID2 committed:
Merge branch 'main' into adding-screenshot

CBID2 commented:
> @CBID2 - do you work locally? Have you pushed your branch to the remote?

@mchammer01, I used Gitpod to make this PR. How about you post your screenshot in the comment section here and I can replace it myself?

mchammer01 committed:
Merge branch 'main' into adding-screenshot

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md:
```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+   ![Screenshot of a pull request. On the right bottom side, a blue, clicked checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```
```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md:
```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```
Changing the name of the screenshot as per our content model.
```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)
```

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

mchammer01 commented:
@CBID2 - I tried to check out your branch but got into some issues 😞 
I made a couple more updates to the PR via the UI.

Can you delete assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png in your PR, and add this screenshot instead, to the same location (note that the name of the new screenshot is different, and that I already updated the reference to it in the article:

The new name is allow-edits-and-access-by-maintainers.png

<img width="364" alt="allow-edits-and-access-by-maintainers" src="https://github.com/github/docs/assets/42146119/b262aa67-dbab-4d82-b673-a1bc4257760e">

CBID2 committed:
Merge branch 'main' into adding-screenshot

CBID2 committed:
fix: replace my screenshot with one made by maintainer

CBID2 commented:
> @CBID2 - I tried to check out your branch but got into some issues 😞 I made a couple more updates to the PR via the UI.
> 
> Can you delete the old screenshot (`assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png`) in your PR, and add this screenshot instead, to the same location (note that the name of the new screenshot is different, and that I already updated the reference to it in the article):
> 
> The new name is allow-edits-and-access-by-maintainers.png
> 
> <img alt="allow-edits-and-access-by-maintainers" width="364" src="https://private-user-images.githubusercontent.com/42146119/290881790-b262aa67-dbab-4d82-b673-a1bc4257760e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2Nzc0NTcsIm5iZiI6MTcwMjY3NzE1NywicGF0aCI6Ii80MjE0NjExOS8yOTA4ODE3OTAtYjI2MmFhNjctZGJhYi00ZDgyLWI2NzMtYTFiYzQyNTc3NjBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE1VDIxNTIzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhZmE2MmUzMDk2MDhmNDhmN2ExN2ExMmYxYjRhNDFkZTdiNjBmYmNjMjFlMzI0ZGM2YjllZjE3YjQxNmRkMDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-lOqIBjADu6VIT4dNDj3w9Rw_1C0pj3zr1iUzwmLlfU">

Hi @mchammer01! :) I changed the screenshot like you requested! :)

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md:
```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)
```
Adding missing comma
```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, enabled checkbox, labeled "Allow edits and access to secrets by maintainers", is highlighted with an orange outline.](/assets/images/help/pull_requests/allow-edits-and-access-by-maintainers.png)
```

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

mchammer01 reviewed (approved):
Thanks for your patience @CBID2, and for adding the screenshot 💖 
This LGTM :shipit:

CBID2 commented:
> Thanks for your patience @CBID2, and for adding the screenshot 💖 
> 
> This LGTM :shipit: 

Anytime @mchammer01! :)

jakirkham commented:
Thanks all! 🙏

This looks great 😀

CBID2 commented:
> Thanks all! 🙏
> 
> This looks great 😀

Anytime @jakirkham! :)
